### PR TITLE
en,zh: add doc about upgrade tidb across multi kubernetes

### PR DESCRIPTION
**Pull request by KanShiori**
### Which TiDB Operator version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v1.2 (TiDB Operator 1.2 versions)
- [ ] v1.1 (TiDB Operator 1.1 versions)
- [ ] v1.0 (TiDB Operator 1.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from: 
- Other reference link(s): 

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: zh/deploy-tidb-cluster-across-multiple-kubernetes.md [hunk 1]
```diff
@@ -561,6 +561,27 @@ spec:
...
+    kind: TidbCluster
+    # ...
+    spec:
+      pd:
+        version: ${version}
+    ```
+
+   2. 按照上一步，升级其他集群的 PD 版本。
+
+2. 以步骤 1 为例，逐一升级所有集群的 TiFlash、TiKV、Pump、TiDB 和 TiCDC 版本。
```
Comment (DanielZhangQD) [inline zh/deploy-tidb-cluster-across-multiple-kubernetes.md:583]:
```suggestion
2. 以步骤 1 为例，按顺序进行如下升级操作：

    1. 升级所有集群中的 TiFlash （如果集群中部署了 TiFlash）
    2. 升级所有集群中的 TiKV
    3. 升级所有集群中的 Pump （如果集群中部署了 Pump）
    4. 升级所有集群中的 TiKV
    5. 升级所有集群中的 TiCDC （如果集群中部署了 TiCDC）
```

Comment (KanShiori) [reply zh/deploy-tidb-cluster-across-multiple-kubernetes.md]: https://github.com/pingcap/docs-tidb-operator/pull/1440/commits/ec6430960feec20b7f7fd2624a204745ae8fdf26

**Review by DanielZhangQD — APPROVED** [review]
LGTM

File: zh/deploy-tidb-cluster-across-multiple-kubernetes.md [hunk 2]
```diff
@@ -561,6 +561,33 @@ spec:
 EOF
 ```
 
+## 升级 TiDB 集群
+
+当跨 Kubernetes 集群部署一个 TiDB 集群时，为了让 TiDB 集群的各组件 Pod 按照特定顺序进行滚动升级，我们需要修改 TidbCluster 定义中组件的 version 配置以控制升级流程。
+
+1. 升级所有集群的 PD 版本。
+
+   1. 修改集群 1 定义中的 `spec.pd.version` 字段，并等待 PD Pod 滚动升级结束。
```
Comment (qiancai) [inline zh/deploy-tidb-cluster-across-multiple-kubernetes.md:570]: 修改完保存后， PD Pod 滚动升级会马上开始吗？可能需要多久。

Comment (KanShiori) [reply zh/deploy-tidb-cluster-across-multiple-kubernetes.md:570]: 会立即开始，多久不能确定。

File: zh/deploy-tidb-cluster-across-multiple-kubernetes.md [hunk 3]
```diff
@@ -561,6 +561,33 @@ spec:
 EOF
 ```
 
+## 升级 TiDB 集群
+
+当跨 Kubernetes 集群部署一个 TiDB 集群时，如果要对 TiDB 集群的各组件 Pod 进行滚动升级，请按照本文中的步骤依次修改 TidbCluster 定义中各组件的 version 配置。
```
Comment (qiancai) [inline zh/deploy-tidb-cluster-across-multiple-kubernetes.md:566]:
```suggestion
当跨 Kubernetes 集群部署一个 TiDB 集群时，如果要对 TiDB 集群的各组件 Pod 进行滚动升级，请按照本文中的步骤依次修改各组件 TidbCluster 定义中的 version 配置。
```

File: zh/deploy-tidb-cluster-across-multiple-kubernetes.md (hunk 2 above)
Comment (qiancai) [reply zh/deploy-tidb-cluster-across-multiple-kubernetes.md:570]: 那如何判断滚动升级是否结束呢？ 需要手动查看状态吗？

Comment (KanShiori) [reply zh/deploy-tidb-cluster-across-multiple-kubernetes.md:570]: 是的

File: zh/deploy-tidb-cluster-across-multiple-kubernetes.md (hunk 3 above)
Comment (KanShiori) [reply zh/deploy-tidb-cluster-across-multiple-kubernetes.md:566]: 这样描述感觉不太准确 🤔 
现在多集群有多个 TidbCluster 定义，然后每个 TidbCluster 下有各组件的配置

[the remark this answers is not in the document]
Comment (qiancai) [reply zh/deploy-tidb-cluster-across-multiple-kubernetes.md:566]: LGTM

File: zh/deploy-tidb-cluster-across-multiple-kubernetes.md [hunk 4]
```diff
@@ -561,6 +561,34 @@ spec:
 EOF
 ```
 
+## 升级 TiDB 集群
+
+当跨 Kubernetes 集群部署一个 TiDB 集群时，如果要对 TiDB 集群的各组件 Pod 进行滚动升级，请按照本文中的步骤依次修改各集群的 TidbCluster 定义中各组件的 version 配置。
```
Comment (qiancai) [inline zh/deploy-tidb-cluster-across-multiple-kubernetes.md:566]:
```suggestion
当跨 Kubernetes 集群部署一个 TiDB 集群时，如果要对 TiDB 集群的各组件 Pod 进行滚动升级，请按照本文中的步骤依次修改各 Kubernetes 集群的 TidbCluster 定义中各组件的 version 配置。
```

File: zh/deploy-tidb-cluster-across-multiple-kubernetes.md [hunk 5]
```diff
@@ -561,6 +561,34 @@ spec:
...
+
+1. 升级所有集群的 PD 版本。
```
Comment (qiancai) [inline zh/deploy-tidb-cluster-across-multiple-kubernetes.md:568]:
```suggestion
1. 升级所有 Kubernetes 集群的 PD 版本。
```

File: zh/deploy-tidb-cluster-across-multiple-kubernetes.md [hunk 6]
```diff
@@ -561,6 +561,34 @@ spec:
...
+   2. 查看 PD Pods 状态，等待集群 1 中的 PD Pod 都重建完毕进入 `Running` 状态。
+
+   3. 按照前两步，升级其他集群的 PD 版本。
+
+2. 以步骤 1 为例，按顺序进行如下升级操作：
+   1. 如果集群中部署了 TiFlash，为所有部署了 TiFlash 的集群升级 TiFlash 版本。
+   2. 升级所有集群中的 TiKV 版本。
+   3. 如果集群中部署了 Pump，为所有部署了 Pump 的集群升级 Pump 版本。
+   4. 升级所有集群中的 TiDB 版本。
+   5. 如果集群中部署了 TiCDC，为所有部署了 TiCDC 的集群升级 TiCDC 版本。
```
Comment (qiancai) [inline zh/deploy-tidb-cluster-across-multiple-kubernetes.md:590]:
```suggestion
   1. 如果集群中部署了 TiFlash，为所有部署了 TiFlash 的 Kubernetes 集群升级 TiFlash 版本。
   2. 升级所有 Kubernetes 集群的 TiKV 版本。
   3. 如果集群中部署了 Pump，为所有部署了 Pump 的 Kubernetes 集群升级 Pump 版本。
   4. 升级所有 Kubernetes 集群的 TiDB 版本。
   5. 如果集群中部署了 TiCDC，为所有部署了 TiCDC 的 Kubernetes 集群升级 TiCDC 版本。
```

File: en/deploy-tidb-cluster-across-multiple-kubernetes.md [hunk 7]
```diff
@@ -566,6 +566,34 @@ spec:
 EOF
 ```
 
+## Upgrade TiDB Cluster
+
+When deploying one TiDB cluster across Kubernetes clusters, in order to upgrade every component Pod in a specific order, we need to modify the version of every component in TidbCluster spec to control the upgrade process.
```
Comment (qiancai) [inline en/deploy-tidb-cluster-across-multiple-kubernetes.md:571]:
```suggestion
For a TiDB cluster deployed across Kubernetes clusters, to perform a rolling upgrade for each component Pod of the TiDB cluster, take the following steps in sequence to modify the version configuration of each component in the TidbCluster spec for each Kubernetes cluster.
```

File: zh/deploy-tidb-cluster-across-multiple-kubernetes.md [hunk 8]
```diff
@@ -561,6 +561,34 @@ spec:
...
+      spec:
+        pd:
+          version: ${version}
+      ```
+    
+   2. 查看 PD Pods 状态，等待集群 1 中的 PD Pod 都重建完毕进入 `Running` 状态。
+
+   3. 按照前两步，升级其他集群的 PD 版本。
+
+2. 以步骤 1 为例，按顺序进行如下升级操作：
```
Comment (qiancai) [inline zh/deploy-tidb-cluster-across-multiple-kubernetes.md:585]:
```suggestion
2. 以步骤 1 为例，按顺序进行如下升级操作：

```

File: en/deploy-tidb-cluster-across-multiple-kubernetes.md [hunk 9]
```diff
@@ -566,6 +566,34 @@ spec:
...
+
+1. Upgrade PD version in all clusters.
```
Comment (qiancai) [inline en/deploy-tidb-cluster-across-multiple-kubernetes.md:573]:
```suggestion
1. Upgrade PD versions for all Kubernetes clusters.
```

File: en/deploy-tidb-cluster-across-multiple-kubernetes.md [hunk 10]
```diff
@@ -566,6 +566,34 @@ spec:
...
+
+   1. Modify `spec.pd.version` field in cluster #1 spec.
```
Comment (qiancai) [inline en/deploy-tidb-cluster-across-multiple-kubernetes.md:575]:
```suggestion
   1. Modify the `spec.pd.version` field in the spec for cluster #1.
```

File: en/deploy-tidb-cluster-across-multiple-kubernetes.md [hunk 11]
```diff
@@ -566,6 +566,34 @@ spec:
...
+      ```yaml
+      apiVersion: pingcap.com/v1alpha1
+      kind: TidbCluster
+      # ...
+      spec:
+        pd:
+          version: ${version}
+      ```
+
+    2. Watch PD Pods, wait for PD Pods in cluster #1 finish rebuilding and become `Running`.
```
Comment (qiancai) [inline en/deploy-tidb-cluster-across-multiple-kubernetes.md:586]:
```suggestion
    2. Watch the status of PD Pods. Wait for PD Pods in cluster #1 to finish recreation and become `Running`.
```

File: en/deploy-tidb-cluster-across-multiple-kubernetes.md [hunk 12]
```diff
@@ -566,6 +566,34 @@ spec:
...
+
+    3. Follow the previous step, upgrade all PD Pods in other clusters.
```
Comment (qiancai) [inline en/deploy-tidb-cluster-across-multiple-kubernetes.md:588]:
```suggestion
    3. Repeat the first two substeps to upgrade all PD Pods in other clusters.
```

File: en/deploy-tidb-cluster-across-multiple-kubernetes.md [hunk 13]
```diff
@@ -566,6 +566,34 @@ spec:
...
+
+2. Take step 1 as an example, perform the following upgrade operations in sequence:
```
Comment (qiancai) [inline en/deploy-tidb-cluster-across-multiple-kubernetes.md:590]:
```suggestion
2. Take step 1 as an example, perform the following upgrade operations in sequence:

```

File: en/deploy-tidb-cluster-across-multiple-kubernetes.md [hunk 14]
```diff
@@ -566,6 +566,34 @@ spec:
...
+    1. Upgrade TiFlash in all clusters, if it is deployed.
+    2. Upgrade TiKV in all clusters.
+    3. Upgrade Pump in all clusters, if it is deployed.
+    4. Upgrade TiDB in all clusters.
+    5. Upgrade TiCDC in all clusters, if it is deployed.
```
Comment (qiancai) [inline en/deploy-tidb-cluster-across-multiple-kubernetes.md:595]:
```suggestion
    1. If TiFlash is deployed in clusters, upgrade the TiFlash versions for all the Kubernetes clusters that have TiFlash deployed.
    2. Upgrade TiKV versions for all Kubernetes clusters.
    3. If Pump is deployed in clusters, upgrade the Pump versions for all the Kubernetes clusters that have Pump deployed.
    4. Upgrade TiDM versions for all Kubernetes clusters.
    5. If TiCDC is deployed in clusters, upgrade the TiCDC versions for all the Kubernetes clusters that have TiCDC deployed.
```

**Review by qiancai — APPROVED** [review]

**Pull request merged**